spread.yaml

```diff
@@ -61,7 +61,7 @@ backends:
                 kernel: GRUB 2
                 workers: 4
             - fedora-25-64:
-                workers: 1
+                workers: 2
```

Comment:
Why do we bump the number of workers to two? @niemeyer requested this to be a single one for the time being. Anything has changed since then?